modules/appeals_api/app/controllers/concerns/appeals_api/character_handling.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  module CharacterHandling
+    extend ActiveSupport::Concern
+
+    WINDOWS_1252_PATTERN = /[^\u0000-\u0255]+/.freeze
+    CENTRAL_MAIL_PATTERN = %r{[^a-zA-Z\-\/\s]}.freeze
+
+    included do
+      def validate_characters
+        characters = request_headers.to_s + params.to_s
+        characters.delete! '"'
+
+        render_invalid_characters_error if characters =~ WINDOWS_1252_PATTERN
+      end
+
+      def transliterate_name(str)
+        I18n.transliterate(str.to_s).gsub(CENTRAL_MAIL_PATTERN, '').strip.first(50)
```

Comment:
No need to extract the CENTRAL_MAIL_PATTERN into a constant if it's only going to be used here.

Reply:
I did think it helped clear up what the heck it is. Maybe just set it as a variable, or forget it?